lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -124,6 +123,75 @@ def setup_students_and_grades(context):
                     )
 
 
+class TestAdminAccessCoachDashboard(CcxTestCase, LoginEnrollmentTestCase):
```

Comment:
@macdiesel  yes, there are tests for verification that active is coach on ccx 
- https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/tests/test_access.py#L117
- https://github.com/edx/edx-platform/blob/master/lms/djangoapps/ccx/tests/test_views.py#L176
- https://github.com/edx/edx-platform/blob/master/lms/djangoapps/ccx/tests/test_views.py#L909
-